samples/fsharp/getting-started/BinaryClassification_SpamDetection/README.md

```diff
@@ -88,15 +87,15 @@ After the model is trained, you can use the `Predict()` API to predict whether n
 let classify = classifyWithThreshold 0.15f
```

Comment:
We don't need to use threshold value.  The model predicts correctly if you follow the pipeline at https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/getting-started/BinaryClassification_SpamDetection/SpamDetectionConsoleApp/Program.cs